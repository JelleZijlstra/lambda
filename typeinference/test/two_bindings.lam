match (1, 2) with {
	(x, y) -> (print x; print y)
};
match ((1, 2), (3,4)) with {
	((x, y), (a, b)) -> (print x; print y; print a; print b)
}
