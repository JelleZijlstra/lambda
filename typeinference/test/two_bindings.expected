%%stdout%%
1
2
1
2
3
4
Result: module
	let __result__ = ()
end
%%stderr%%
